css/sitesummary.css

```diff
@@ -11,3 +11,25 @@
 .gridfield-button-link {
     margin-bottom: 12px;
 }
+
+.cms table.ss-gridfield-table tr td.col-Summary {
+    padding: 0;
+}
+
+a.package-summary__anchor {
```

Comment:
Can you please remove the element type from these selectors? The class name should be enough

Reply:
Because of the `inherit` definitions for colour and the underlining I need to be more specific 😧 . I can take out those attributes and make the text actually look like links?

Reply:
Nah, all good